packages/components/pie-text-input/src/index.ts

```diff
@@ -218,12 +219,15 @@ export class PieTextInput extends FormControlMixin(RtlMixin(LitElement)) impleme
                     placeholder=${ifDefined(placeholder)}
                     ?readonly=${readonly}
                     ?required=${required}
+                    aria-describedby=${ifDefined(assistiveText ? assistiveTextIdValue : undefined)}
+                    aria-invalid=${status === 'error' ? 'true' : 'false'}
+                    aria-errormessage="${ifDefined(status === 'error' ? assistiveTextIdValue : undefined)}"
```

Comment:
Currently the `aria-errormessage` will exist even if `assistiveText` does not exist - should we prevent this?